ramls/samples/fieldName.sample

```diff
@@ -0,0 +1,7 @@
+{
+   "id":"instance.identifiers.invalid.issn",
+   "displayNameKey":"instance.identifiers",
+   "referenceDataValue":"Invalid ISSN",
```

Comment:
I thought we are going to use a dot formatted name here how will the UI translate this value?